ivy/functional/frontends/paddle/nn/functional/loss.py

```diff
@@ -428,6 +428,16 @@ def softmax_with_cross_entropy(
     return paddle.to_tensor(loss)
 
 
+@handle_exceptions
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")
+def square_error_cost(input, label, name=None):
```

Comment:
Hey, there is another function in line 443 with the same name. Why is this implemented again? Let me know if I missed something.